packages/vscode-dependencies-validation/.mocharc.js

```diff
@@ -0,0 +1,8 @@
+const chai = require("chai");
+const chaiAsPromised = require("chai-as-promised");
```

Comment:
evaluate option of having a root configuration for mocha.
and maybe customizing (extending it) per specific sub-package (if needed).

Reply:
Let's talk about it